README.md

```diff
@@ -0,0 +1,27 @@
+# pacbot Hardware Access Layer
+
+## Features:
+- [x] Drives DC motors over an H-Bridge with PWM
+- [x] Reads high frequency quadrature signals
+- [ ] Configures and reads from multiple VL6180X distance sensors
+- [x] Uses the super secret DMP on the MPU6050
```

Comment:
This is untrue

Reply:
yeah the uses the super secret dmp was not checked

Reply:
no, I'm saying we _don't_ use the DMP

Reply:
but in that branch it does though, or at least there was a driver for it